translation/translation.js

```diff
@@ -0,0 +1,271 @@
+/*This javascript script translate (in this case from italian to french) layers and groups names, the title of the project, 
+the print layout name and the aliases or fileds names according to the language of the browser and using the json file created
+with the python script get_translatable_string.py
+If a translated string is provided in the json file the original string is translated, otherwise the original string is shown.
+The json file must be saved in the media folder which need to be available from the web. To do this it is necessary to create 
+a symbolic link on the apache directory (e.g. /var/www/html/) to the media folder in the user repository.*/
+
+//This function is executed when the uicreated event is triggered
+lizMap.events.on({
+    uicreated: function(e) {
+        // get browser language
+        var userLang = navigator.language || navigator.userLanguage;
+        if (userLang === 'fr' || userLang === 'fr-FR'){      
+            
+            var translationIT_FR = {};
+
+            // get the json file with transalted strings
+            // the path to the json file must be provided as relative path, using the web url the browser can return a security error
+            fetch('../../../../../../concerteaux3d/translation.json').then(function(response) {
```

Comment:
Could you put the URL in a variable at top of the script ?

Reply:
We should use Lizmap Web Client media capabilities here I think. If lizmap is accessible via
https://demo.lizmap.com/lizmap/index.php
you can fetch for example the file `media/photo/beluga.jpg` build an URL like this one 
https://demo.lizmap.com/lizmap/index.php/view/media/getMedia?repository=demo&project=cats&path=media/photo/beluga.jpg

To build this URL, you can use 

```javascript
var mediaUrl = OpenLayers.Util.urlAppend(
     lizUrls.media,
     OpenLayers.Util.getParameterString({
         "repository": lizUrls.params.repository,
         "project": lizUrls.params.project,
         "path": "media/photo/beluga.jpg"
     })
);
```

Reply:
I have implemented the solution suggested by @mdouchin and it works without the symbolic link on the apache directory.